Tests/MongoSwiftTests/MongoCollectionTests.swift

```diff
@@ -105,9 +105,38 @@ final class MongoCollectionTests: MongoSwiftTestCase {
     }
 
     func testDrop() throws {
-        expect(try self.coll.drop()).toNot(throwError())
-        // insert something so we don't error when trying to drop again in teardown
-        expect(try self.coll.insertOne(self.doc1)).toNot(throwError())
+        let encoder = BSONEncoder()
+        let center = NotificationCenter.default
+
+        let client = try MongoClient(options: ClientOptions(eventMonitoring: true))
+        client.enableMonitoring(forEvents: .commandMonitoring)
+
+        var db = client.db(type(of: self).testDatabase)
+        let writeConcern = try WriteConcern(journal: true, w: .number(1))
```

Comment:
`writeConcern` here is identical to `expectedWriteConcern`, so we can remove it and just use `expectedWriteConcern`.